core/src/main/java/io/questdb/griffin/engine/table/LatestByValueDeferredIndexedRowCursorFactory.java

```diff
@@ -67,6 +63,7 @@ public RowCursor getCursor(DataFrame dataFrame) {
     @Override
     public void prepareCursor(TableReader tableReader, SqlExecutionContext sqlExecutionContext) {
         if (symbolKey == SymbolTable.VALUE_NOT_FOUND) {
```

Comment:
I believe this code will cache `symbolKey` and never re-evaluate it which was fine before using with bind variables. Now this `if` should go away.

Reply:
Nice catch. Fixed in 3fcfb1a9977e3a79b52a5e289a70e7754f0bba34